debian/copyright

```diff
@@ -1,7 +1,29 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: mesaflash
+Upstream-Contact: LinuxCNC Developers <emc-developers@lists.sourceforge.net>
 Source: https://github.com/micges/mesaflash
 
+Files: *
 Copyright: 2013-2014 Michael Geszkiewicz
 License: GPL-2+
```

Comment:
Does this also need to be corrected to say "GPL-2.0+"?

Reply:
Yes, please - forgotten --amend on my side.